include/repeated_hafnian.hpp

```diff
@@ -41,70 +97,92 @@ namespace libwalrus {
  */
 template <typename T>
 inline T hafnian_rpt(std::vector<T> &mat, std::vector<int> &rpt) {
-    int n = std::sqrt(static_cast<double>(mat.size()));
-    assert(static_cast<int>(rpt.size()) == n);
+  int n = std::sqrt(static_cast<double>(mat.size()));
+  assert(static_cast<int>(rpt.size()) == n);
+  T y = 0.0;
 
-    long double p = 2;
-    T y = 0.0, q = 0.0;
+#pragma omp parallel
+  {
+    T q = 0.0L;
 
-    std::vector<int> x(n, 0.0);
+    long double p = 2;
+    // std::vector<int> x(n, 0.0);
```

Comment:
How come this is commented out? If it's no longer needed, can it be deleted?

Reply:
thanks deleted